--- protobuf-config.cmake.orig	2023-01-25 15:33:56.202168989 -0500
+++ protobuf-config.cmake	2023-01-25 15:34:19.058513933 -0500
@@ -146,9 +146,9 @@ function(protobuf_generate)
 
     add_custom_command(
       OUTPUT ${_generated_srcs}
-      COMMAND protobuf::protoc
+      COMMAND ${PROTOBUF_PROTOC_EXEC}
       ARGS ${protobuf_generate_PROTOC_OPTIONS} --${protobuf_generate_LANGUAGE}_out ${_plugin_options}:${protobuf_generate_PROTOC_OUT_DIR} ${_plugin} ${_protobuf_include_path} ${_abs_file}
-      DEPENDS ${_abs_file} protobuf::protoc
+      DEPENDS ${_abs_file} ${PROTOBUF_PROTOC_EXEC}
       COMMENT ${_comment}
       VERBATIM )
   endforeach()
