diff --git a/CMakeLists.txt b/CMakeLists.txt
index 20a638d..fa376ec 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -141,7 +141,7 @@ add_subdirectory(docs)
 ########################################################################
 # SWIG wrappers (optional)
 ########################################################################
-add_subdirectory(swig)
+#add_subdirectory(swig)
 
 ########################################################################
 # uninstall target
